### PR TITLE
raspivid: Fix so you can record raw without encoding too.

### PR DESCRIPTION
**Pull request by 6by9**
You could use raspividyuv to do it, but there's no reason to
prevent doing it in raspivid.

Just to make @gsh happy.

### CODE REVIEW

File: host_applications/linux/apps/raspicam/RaspiVid.c [hunk 1]
```diff
@@ -2937,7 +2938,7 @@ int main(int argc, const char **argv)
                }
 
                // Send all the buffers to the splitter output port
-               if (state.raw_output) {
+               if (state.callback_data.raw_file_handle) {
                   int num = mmal_queue_length(state.splitter_pool->queue);
```
Comment (JamesH65) [inline host_applications/linux/apps/raspicam/RaspiVid.c]: Is the raw_output flag still requried elsewhere?

Comment (6by9) [reply host_applications/linux/apps/raspicam/RaspiVid.c]: I don't believe so.
It's used for setup, and nuked if the file handle fails to open. The rest uses the file handle.

**Comment by 6by9** [comment]
@JamesH65 Any further comments?

File: host_applications/linux/apps/raspicam/RaspiVid.c (hunk 1 above)
Comment (JamesH65) [reply host_applications/linux/apps/raspicam/RaspiVid.c]: Should probably remove it then, but TBH, hardly worth the effort. Othewise LGTM.

Comment (6by9) [reply host_applications/linux/apps/raspicam/RaspiVid.c]: Sorry, misintepreted your comment.
The command line parser sets raw_output. The setup code then checks it and will open the file and assign callback_data.raw_file_handle.
The rest of the code then only looks at callback_data.raw_file_handle.

**Pull request merged**